products/batch_exports/backend/temporal/destinations/snowflake_batch_export.py

```diff
@@ -580,6 +581,8 @@ async def copy_loaded_files_to_snowflake_table(
         self,
         table_name: str,
         table_stage_prefix: str,
+        table_fields: list[SnowflakeField],
+        known_variant_columns: list[str],
```

Comment:
we need `known_variant_columns` because `elements` is also a `VARIANT`, although its contents is often not valid JSON (at least in our automated tests). I'm not sure if it should be a `VARIANT` though because in BigQuery for example it is a `STRING`